portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsDateField.testcase

```diff
@@ -31,6 +31,144 @@ definition {
 		}
 	}
 
+	@description = "Verify that is able to add a future validation using the response date as parameter"
+	@priority = "5"
+	test CanAddFutureValidationUsingResponseDate {
+		property forms.functional = "true";
+
+		var displayDateDay = DateUtil.getCurrentDate();
+		var displayDateMonth = DateUtil.getCurrentMonth();
+		var displayDateYear = DateUtil.getCurrentYear();
+		var futureYear = ${displayDateYear} + 1;
+		var pastYear = ${displayDateYear} - 1;
+		var displayFutureDate = "${displayDateMonth}/${displayDateDay}/${futureYear}";
+		var displayPastDate = "${displayDateMonth}/${displayDateDay}/${pastYear}";
+		var futureHalfYear = StringUtil.extractLast("${futureYear}","0");
+
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Date");
+
+		Form.editFieldLabelMetal(fieldValue = "Date Field");
+
+		Form.gotoAdvancedTab();
+
+		FormFields.enableSwitch(fieldName = "Validation");
+
+		FormFields.validationRangeCustomDate(acceptedDateOption = "Future Dates");
+
+		FormFields.validationCustomDateStartsFrom(
+			startsFromOperation = "Plus",
+			startsFromOption = "Custom Date",
+			startsFromQuantity = "3",
+			startsFromUnit = "Days");
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.viewLabel(
+			fieldLabel = "Date Field",
+			fieldName = "Date Field");
+
+		FormViewBuilder.editDate(
+			fieldName = "Date Field",
+			fieldValue = "${displayPastDate}");
+
+		FormPortlet.submitWithValidationError(validationErrorMessage = "This field is invalid.");
+
+		FormViewBuilder.editDate(
+			fieldName = "Date Field",
+			fieldValue = "${displayFutureDate}");
+
+		FormPortlet.submitSuccessfully();
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableEntry(
+			fieldLabel = "Date Field",
+			fieldValue = "${displayDateMonth}/${displayDateDay}/${futureHalfYear}");
+	}
+
+	@description = "Verify that is able to add a past and range validation using the response date as parameter"
+	@priority = "5"
+	test CanAddPastValidationUsingResponseDate {
+		property forms.functional = "true";
+
+		var displayDateDay = DateUtil.getCurrentDate();
+		var displayDateMonth = DateUtil.getCurrentMonth();
+		var displayDateYear = DateUtil.getCurrentYear();
+		var futureYear = ${displayDateYear} + 1;
+		var pastYear = ${displayDateYear} - 2;
+		var displayFutureDate = "${displayDateMonth}/${displayDateDay}/${futureYear}";
+		var displayPastDate = "${displayDateMonth}/${displayDateDay}/${pastYear}";
+		var pastHalfYear = StringUtil.extractLast("${pastYear}","0");
```

Comment:
@rafaellajordao, can we reuse these variables on all other tests demanding it? If so, please move them to the `setup` method at the beginning of this file.